### PR TITLE
Implement conversion from spherical to Cartesian

### PR DESCRIPTION
**Pull request by matzipan**
Implement s2c and add basic test for issue #2

### CODE REVIEW

File: src/util/Util.jl [hunk 1]
```diff
@@ -214,5 +214,94 @@ function point_on_limb(ellipsoid, observer, outside)
...
+   cp = cos(phi);
+   x = cos(theta) * cp;
+   y = sin(theta) * cp;
+   z = sin(phi);
+
+   return x, y, z
+end
+
+
+"""
```
Comment (helgee) [inline src/util/Util.jl:247]: It should suffice to put this into the LICENSE.md and just put a reference here, no? IANAL and all.

Comment (matzipan) [reply src/util/Util.jl:247]: No clue. Should I do it?

File: src/util/Util.jl [hunk 2]
```diff
@@ -214,5 +214,94 @@ function point_on_limb(ellipsoid, observer, outside)
...
+
+- `theta`: longitude angle in radians
+- `phi`: latitude angle in radians
+
+### Returns ###
+- `x`: magnitude of projection on x axis
+- `y`: magnitude of projection on y axis
+- `z`: magnitude of projection on z axis
+
+Derived, with permission, from the SOFA library.  See notes at end of file.
```
Comment (helgee) [inline src/util/Util.jl]:
```suggestion
### References ###

- [ERFA](https://github.com/liberfa/erfa/blob/master/src/s2c.c)
```

File: test/util.jl [hunk 3]
```diff
@@ -31,4 +31,12 @@ import SPICE
     @testset "plane_section" begin
         xy = plane_section([1.0, 2.0, 3.0], zeros(3), [0.0, 0.0, 1.0])
     end
+    @testset "spherical_to_cartesian" begin
+        act = spherical_to_cartesian(3.0123, -0.999)
+        exp = (-0.5366267667260523906, 0.0697711109765145365, -0.8409302618566214041)
+        @testset for i in eachindex(exp, act)
+            @test isapprox(exp[i], act[i], rtol=0, atol=1e-12)
```
Comment (helgee) [inline test/util.jl]:
```suggestion
            @test exp[i] ≈ act[i] atol=1e-12
```

Comment (helgee) [reply test/util.jl]: You can omit `rtol` if you provide `atol`

Comment (giordano) [reply test/util.jl]: Speaking for myself, usually I prefer `rtol` to `atol`, an absolute error tells nothing about the significance of an error.

Comment (helgee) [reply test/util.jl]: Unless the value is close to zero in which case you should definitely use `atol` 😉

File: src/util/Util.jl (hunk 1 above)
Comment (helgee) [reply src/util/Util.jl:247]: Just remove it for now. I will take care of updating the LICENSE file on `master`.

File: test/util.jl (hunk 3 above)
Comment (giordano) [reply test/util.jl]: Fair enoguh :slightly_smiling_face:

File: src/util/Util.jl (hunk 1 above)
Comment (helgee) [reply src/util/Util.jl:247]: I have asked the ERFA maintainers how we should handle this:https://github.com/liberfa/erfa/issues/67

**Pull request merged**